opentimelineio/adapters/cmx_3600.py

```diff
@@ -104,23 +104,18 @@ def add_clip(self, line, comments, rate=24):
         comment_handler = CommentHandler(comments)
         clip_handler = ClipHandler(line, comment_handler.handled, rate=rate)
         clip = clip_handler.clip
-        reel = clip_handler.reel
-
-        # A reel name of `AX` represents an unknown or auxilary source
-        # We don't currently track these sources outside of this adapter
-        # So lets skip adding AX reels as metadata for now,
-        # as that would dirty json outputs with non-relevant information
-        if reel != 'AX':
-            clip.metadata.setdefault("cmx_3600", {})
-            clip.metadata['cmx_3600']['reel'] = reel
-
         if comment_handler.unhandled:
             clip.metadata.setdefault("cmx_3600", {})
             clip.metadata['cmx_3600'].setdefault("comments", [])
             clip.metadata['cmx_3600']['comments'] += (
                 comment_handler.unhandled
             )
 
+        # Add reel name to metadata
+        if clip_handler.reel:
```

Comment:
So this is preserving even the 'AX' cases that @mikekoetter had pruned.  Do you think the strategy to ignore the AX ones is better?  (see the comment in the deleted block).

Reply:
Yeah, when I look at it I see no reason for adding AX to the metadata. It should still be there in the event line if no clip is present.
I'll add the test for it.

Reply:
If you add the test, can you add back the comment?  I thought that was a good explanation.

Reply:
@ssteinbach I added the pruning of 'AX', but and issue with the `test_cdl.test_cdl_round_trip` might have reviled some poor design choices from my end. I'm going to dig a bit deeper to come up with a solution, but I'm afraid I need until next week as I need to sleep now and I'm away from my computer this weekend. Sorry about that.

Reply:
> If you add the test, can you add back the comment? I thought that was a good explanation.

Sure, no problem.

Reply:
No rush!  Definitely get your rest!